<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modificar_Dados</class>
 <widget class="QWidget" name="Modificar_Dados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>370</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(0, 170, 255);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>350</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="9" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font-weight:bold;
font-size:13px;</string>
            </property>
            <property name="text">
             <string>E-mail:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_dadosemail1">
              <property name="minimumSize">
               <size>
                <width>290</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>290</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:white;
border-radius: 6px;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="11" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-weight:bold;
font-size:13px;</string>
            </property>
            <property name="text">
             <string>Confirmar e-mail:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QLineEdit" name="line_dadosemail2">
              <property name="minimumSize">
               <size>
                <width>290</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>290</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:white;
border-radius: 6px;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">
font-weight:bold;
font-size: 18px;</string>
          </property>
          <property name="text">
           <string>Modificar dados de Cadastro</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Se não deseja modificar ambos os campos basta modificar somente o que for necessário e deixar o outro campo do jeito que ele está (campos de e-mail em branco e comboBox na opção pré-estabelecida baseado no seu atual cargo registrado)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font-weight:bold;
font-size:13px;</string>
            </property>
            <property name="text">
             <string>Cargo:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QComboBox" name="combo_dadoscargos">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius: 6px;
background-color: white;</string>
              </property>
              <item>
               <property name="text">
                <string>Desenvolvedor Jr.</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Desenvolvedor Sr.</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Admin</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="button_dados_enviarmod">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(185, 190, 217);
	color: black;
	font-weight: 700;
	border-radius: 6px;
}
QPushButton:hover{
	background-color: white;
}</string>
            </property>
            <property name="text">
             <string>Enviar Modificações</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 10px;
color: red;</string>
          </property>
          <property name="text">
           <string>Obs: Caso decida modificar o cargo de registro para Admin, você será removido automaticamente da empresa em que possui registro (se possuir) e será necessário iniciar sua própria empresa dependendo do novo cargo escolhido. E caso já seja Admin e queira modificar para um cargo de Desenvolvedor, sua empresa será removida do banco de dados.</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
